twoopstracker/twoops/views.py

```diff
@@ -231,8 +231,11 @@ def get_queryset(self):
         user = self.request.user
 
         if user.is_authenticated:
+            accounts_groups = user.userprofile.account_groups.all()
             twitter_accounts_lists = TwitterAccountsList.objects.filter(
-                Q(is_private=False) | Q(owner=user.userprofile)
+                Q(is_private=False)
+                | Q(owner=user.userprofile)
+                | Q(account_groups__in=accounts_groups)
```

Comment:
Just thinking out loud: can't we combine getting accounts_groups and filter in one? Sort of like:
```python
            twitter_accounts_lists = TwitterAccountsList.objects.filter(
                Q(is_private=False)
                | Q(owner=user.userprofile)
                | Q(account_groups__members__id=user.userprofile.id)
```

Reply:
Sure. It's possible.